target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -66,6 +69,11 @@ mediatek_setup_macs()
 	bananapi,bpi-r3)
 		wan_mac=$(macaddr_add $(cat /sys/class/net/eth0/address) 1)
 		;;
+	h3c,magic-nx30-pro)
+		lan_mac=$(mtd_get_mac_ascii pdt_data_1 ethaddr)
+		wan_mac=$(macaddr_add "$lan_mac" 1)
+		label_mac=$lan_mac
+		;;
```

Comment:
`$(mtd_get_mac_ascii pdt_data_1 ethaddr)` should be used as `$wan_mac` and `$label_mac` to keep compatible with stock firmware.

Reply:
Thanks, I have re-checked this in the stock firmware.